lib/s3-multipart.js

```diff
@@ -632,8 +632,62 @@ function s3UploadPartHandler() {
 
                     // Return ETag header (required by S3 clients)
                     res.setHeader('ETag', '"' + finalETag + '"');
-                    res.send(200);
-                    next(false);
+
```

Comment:
Is it just me or are all replies in s3UploadPartHandler() now going to perform CORS ?  Or am I missing (maybe from new files or large-changes I've not yet read yet) something?